doc/tutorial/index.rst

```diff
@@ -184,6 +195,243 @@ as before, or leverage the convenience script as follows:
 After running this command, you will see that ``index.html`` reflects the new
 changes!
 
+Exporting your documentation to other formats
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sphinx supports a variety of formats apart from HTML, including PDF, EPUB,
+:ref:`and more <builders>`.  For example, to build your documentation as an
+e-book in EPUB format, run this command from the ``docs`` directory:
+
+.. code-block:: console
+
+   (.venv) $ make epub
```

Comment:
Is there a reason we're doing epub here instead of `pseudoxml`? Just ease of browsing?

Reply:
Yes, I went back to your original suggestion of using EPUB, because it's a more familiar format. For beginners, it's unclear what the `pseudoxml` builder is for, and in fact:

> This builder is used for debugging the Sphinx/Docutils “Reader to Transform to Writer” pipeline.

https://www.sphinx-doc.org/en/master/usage/builders/index.html#sphinx.builders.xml.PseudoXMLBuilder

The good thing is that, even if the user doesn't have an EPUB viewer, they can still open the index in a web browser.